packages/html/ds/src/textarea/textarea.schema.ts

```diff
@@ -0,0 +1,33 @@
+import * as zod from 'zod';
+
+export const textAreaSchema = zod.object({
+  hasError: zod
```

Comment:
I think the `hasError` can be transformed in a `withError` that accepts a `string` . That way you can integrate both the label of the error and the boolean value of it

Reply:
I agree